index.js

```diff
@@ -5650,10 +5654,10 @@ module.exports = {
                 },
                 'q-applicant-physical-injuries-arms-shoulder-other': {
                     type: 'string',
-                    title: 'Type of injury',
+                    title: 'Other shoulder injuries',
```

Comment:
this question and the torso-shoulder question have the same descriptors (6 in total). Is the descriptor `Other shoulder injuries` OK for both questions?

Reply:
I'll flag this with Julie but this content is in line with the data dictionary.